locust/input_events.py

```diff
@@ -94,6 +94,7 @@ def input_listener_func():
                     gevent.sleep(0.2)
         except Exception as e:
             logging.warning(f"Exception in keyboard input poller: {e}")
+            raise
```

Comment:
I've added this for a more robust code, but I'm not sure if there are any valid exceptions that could happen during a normal session. If so, we should probably handle them seprately.